name = "throw";
id = "throw"
alias[] = ""
description = "Throw is the keyword which raise an exception. 

When an exception is thrown, the normal flow of the program is interrupted, and PHP starts looking for an exception handler to catch and handle the exception."
code = "<?php
try{
    throw new Exception('Error');
} catch (\\Exception $e) {
    print 'Warning : error was identified!';
}

?>"
documentation = "https://www.php.net/manual/en/language.exceptions.php"
analyzers[] = "Exceptions/ThrowFunctioncall"
analyzers[] = "Php/ThrowWasAnExpression"
analyzers[] = "Php/ThrowUsage"
cobblers[] = ""
phpVersionSince = ""
phpVersionUntil = ""
seeAlso[] = "`Exceptions - PHP Advance <https://jobtensor.com/Tutorial/PHP/en/Exceptions>`_"
related[] = "try-catch"
related[] = "exception"
phpVersionDeprecated = "";
popularity = "";
rule = "";
packagist[] = "";
tags[] = "";
